prboom2/src/p_plats.c

```diff
@@ -156,7 +159,11 @@ void T_PlatRaise(plat_t* plat)
           {
             case raiseAndChange:
             case raiseToNearestAndChange:
-              P_RemoveActivePlat(plat);
```

Comment:
Actually, @SmileTheory 's patch for Choco only affects the `case up` code path and the `case down` code path should already be disabled in doom_12_compatibility because of the `if (!comp[comp_floors])` guard, right?

Reply:
You are right, I missed the comp_floors guard.
You can train a monkey to submit PRs but no amount of bananas will turn it into a programmer. ;)

Reply:
Thanks! 

Still, the crucial question remains: Do we want to develop complevel 0 to become more accurate for the sake of demo compatibility with Vanilla Doom 1.2? Or do we want to keep it as it is with known bugs for the sake of demo compatibility with itself?

In Woof we faced a similar question and decided to go the second route, i.e. we replaced original MBF code with the pendant from PrBoom+ in order to remain demo compatible with complevel 9.

Reply:
> Do we want to develop complevel 0 to become more accurate for the sake of demo compatibility with Vanilla Doom 1.2? Or do we want to keep it as it is with known bugs for the sake of demo compatibility with itself?

I overlooked your comment during the holiday season. These are important questions to consider. 

Fortunately, the issue is clear here. The number of -cl0 demos recorded with prboom-plus is insignificant compared to that of Boom- and MBF-format demos - or even to the existing body of v1.2 demos recorded with vanilla - and of those -cl0 demos currently only two or three are known to be broken by improvements in v1.2 emulation. 

So there's no case to make for a "prboom -cl0 standard", this is just a case of a handful broken demos. The will be marked accordingly on DSDA (one or two of mine already are), and perfecting v1.2 emulation should go on unhindered.

These changes may indeed present a significant issue in DSDA-Doom, used to record dozens of Heretic -cl0 demos on DSDA. Should that happen I'm quite confident that kraflab is more than up to the task of coming up with a solution both elegant and efficient.

Reply:
There's no issue for dsda-doom. As I posted in my comment, this logic is already happening in dsda-doom, but only for heretic. I am concerned though if this patch is completely correct, since it _does_ have different cases than heretic's code. It implies heretic patched this switch, but only for some of the plat types? 🤔